CMakeLists.txt

```diff
@@ -76,6 +76,9 @@ option(BUILD_CURL_EXE "Set to ON to build curl executable." ON)
 option(CURL_STATICLIB "Set to ON to build libcurl with static linking." OFF)
 option(ENABLE_ARES "Set to ON to enable c-ares support" OFF)
 if(WIN32)
+  option(CURL_STATIC_CRT "Set to ON to build libcurl with static CRT on Windows (/MT)." OFF)
+  option(ENABLE_INET_PTON "Set to OFF to prevent usage of inet_pton on Windows. For compatability with Windows XP, Windows Server 2003 etc." ON)
```

Comment:
Why a specific `ENABLE_INET_PTON` option? Are you building on a system version that *has* the function but you want it to run on older ones? Can we somehow get that message into the explanation for its purpose?